static/src/js/components/SpatialSelection/SpatialSelection.js

```diff
@@ -99,6 +99,9 @@ const SpatialSelection = (props) => {
   const drawControl = useRef(null)
   const [preEditBounds, setPreEditBounds] = useState(null)
   const featureGroupRef = useRef(null)
+  // https://stackoverflow.com/questions/57847594/accessing-up-to-date-state-from-within-a-callback/62453660#62453660
+  const stateRefPreEditBounds = useRef()
```

Comment:
We should try to fix this with `useCallback` useRef outside the context of manipulation DOM elements is not preferred

Reply:
We discussed this further didn't seem to be an alternative way without potentially significantly more refactoring here to do this. useRef is already being used and currently this block isn't actually running due to hte bug